resources/real_opus.yaml

```diff
@@ -2655,3 +2655,45 @@ action_templates:
 assets:
   "script":
     path: "Apornas o 2021-11-22.pdf"
+ui:
+  shortcuts:
```

Comment:
While you're at it, could you add a shortcut for the "tekniska problem" achievement? When it's applicable, we may not want to spend time finding it in the inventory tab.

Reply:
Will do!